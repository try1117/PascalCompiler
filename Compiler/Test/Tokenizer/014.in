0.12987E0