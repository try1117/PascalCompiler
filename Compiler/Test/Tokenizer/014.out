0  | 0  | CONST_DOUBLE             | 0.12987E0                | 0.129870000000000
